.github/pull_request_template.md

```diff
@@ -0,0 +1,14 @@
+## Description
+
+What changed? Why? (the code + comments should speak for itself on the "how")
+
+Fun screenshots ir a cool video or something are super helpful as well. If this touches platform-specific behavior, this is where test evidence should be collected.
+
+## Meta
+
+Merge checklist:
+- [ ] Pull Request title is [short, imperitive summary](https://cbea.ms/git-commit/) of proposed changes
+- [ ] The description documents the _what_ and _why_
+- [ ] If this PR touches photon-serde, all messages have been regenerated and hashes have not changed unexpectedly
```

Comment:
Can't we just check this in CI

Reply:
No, but we should make a GitHub Actions bot drop a comment if the hashes change. There are times when it's fine if they do.